src/Tribe/Views/V2/View.php

```diff
@@ -2223,13 +2254,10 @@ protected function setup_additional_views( array $events = [], array $template_v
 		 * @param array $template_vars An associative array of variables that will be set, and exported, in the template.
 		 * @param View  $instance      The current View object.
 		 */
-		$recent_past_threshold = apply_filters( "tribe_events_views_v2_threshold_to_show_recent_past_events", 0, $events, $template_vars, $this );
-
-		//todo change to get a saved value instead of running the query
-		$upcoming_events_count = tribe_events()->where( 'ends_after', 'now' )->count();
+		$recent_past_threshold = apply_filters( "tribe_events_views_v2_threshold_to_show_recent_past_events", absint( 0 ), $events, $template_vars, $this );
```

Comment:
:no_entry_sign: String "tribe_events_views_v2_threshold_to_show_recent_past_events" does not require double quotes; use single quotes instead

<sub>hash: ef02a26fddc6805cb9e9bf12fad46446</sub>